### ISSUE TITLE
Add support for FTP active mode

### ISSUE THREAD

**Issue by tklew** [mar10/pyftpsync#21]
Hi :
- Due to security concerns, do pyftpsync support active mode for FTP ?
- The FTP server denied passive mode and do not allow pyftpsync to write any lock file ?

Thanks !

```
resp* '230 Login successful.'
*cmd* 'CWD /tmp/b'
*resp* '250 Directory successfully changed.'
*cmd* 'PWD'
*resp* '257 "/tmp/b" is the current directory'
*cmd* 'TYPE I'
*resp* '200 Switching to Binary mode.'
*cmd* 'PASV'
*resp* '550 Permission denied.'
Could not write lock file: 550 Permission denied.
Upload /tmp/a
                  to ftp://localhost/tmp/b
*cmd* 'TYPE A'
*resp* '200 Switching to ASCII mode.'
*cmd* 'PASV'
*resp* '550 Permission denied.'
Skip remove lock file (was not written)
*cmd* 'QUIT'
```

**Comment by mar10** [comment]
We use [ftplib](https://docs.python.org/3/library/ftplib.html), which uses passive mode by default.
There is no option yet, but that should be easy to add.

Btw., are you sure you have write access at all, i.e. can you issue these commands with using a simple ftp client (using active and passive mode)?

**Comment by khubab** [comment]
Yes, pyftpsync does support active mode for FTP. I had also faced this 550 error but it is related to user directory. In my case, the account was pointing to main directory of linux which has .cache folder. Though, the ftp user doesn't have any write permission for this folder and it has generated error of 550. I had realized it and changing the root directory of the ftp user to its actual directory, that's in my case home/ftpuser has fixed the issue. pyftpsync wasn't able to write the json file which was producing the 550 permission error.
